### DISCUSSION TITLE
when i use proxy i got empty cookies array can any one help please

### DISCUSSION THREAD

**Discussion by 0xtyassine** [FlareSolverr/FlareSolverr#1094] [Q&A] [answered]
def FlareBypass(self, data, headers={"Content-Type": "application/json"}):
        response = requests.post("http://localhost:8191/v1", headers=headers, json=data)
        return response
   
    def Flare_sessions_create(self):
        data = {
            "cmd": "sessions.create",
            "session": self._driver.session_id,
            "proxy": {"url": "http://username:password@url:port"}
        }
        return self.FlareBypass(data).json()["session"]
        
    def Flare_get(self, url):
        data = {
            "cmd": "request.get",
            "url": url,
            "maxTimeout": 10000,
            "session": self.Flare_sessions_create(),
            # session_ttl_minutes
            # cookies : "cookies": [{"name": "cookie1", "value": "value1"}, {"name": "cookie2", "value": "value2"}]
            # "returnOnlyCookies" : True,
        }
        try:
            response = self.FlareBypass(data)
            print(response.json())
            return response
        except Exception as e:
            print(e)
            exit(1)

**Comment by ilike2burnthing** [comment, marked as answer]
Use correct proxy format - https://github.com/FlareSolverr/FlareSolverr#-sessionscreate

**Reply by 0xtyassine** [reply]
thank u so much
